Bottom: 9023ede0dfcb282ab90f7726e5c637c9ff842534
Top:    b81754028658b7a877798e12b2fd67dd9d2ae0be
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-06-05 17:05:53 -0700

Refresh of rs-spin

---

diff --git a/src/rsocket.c b/src/rsocket.c
index 90f9a10..f53f1c8 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -66,7 +66,7 @@
 static struct index_map idm;
 static pthread_mutex_t mut = PTHREAD_MUTEX_INITIALIZER;
 
-static uint32_t polling_time;
+static uint32_t polling_time = 10;
 
 /*
  * Immediate data format is determined by the upper bits
